src/components/RegisterLoginModal/RegisterLoginModal.tsx

```diff
@@ -4,17 +4,16 @@ import Button from '@mui/material/Button';
 import IconButton from '@mui/material/IconButton';
 import Modal from '@mui/material/Modal';
 import { styled } from '@mui/material/styles';
+import LoginForm from 'components/LoginForm/LoginForm';
+import RegisterForm from 'components/RegisterForm/RegisterForm';
+import RegisterLoginHeader from 'components/RegisterLoginHeader/RegisterLoginHeader';
```

Comment:
Changing them to:
```
import LoginForm from 'src/components/LoginForm/LoginForm';
import RegisterForm from 'src/components/RegisterForm/RegisterForm';
import RegisterLoginHeader from 'src/components/RegisterLoginHeader/RegisterLoginHeader';
```
will prevent the linter from complaining and the types you specified will be correctly recognised.

Reply:
I had to use relative path :(